### PR TITLE
Change `Notify success event to Slack/Hipchat` option to `Notify all event to Slack/Hipchat`

### PR DESCRIPTION
**Pull request by eisuke**
This PR changes the `Notify success event to Slack/Hipchat` option to `Notify all event to Slack/Hipchat`.

And Kuroko2 notifies launch/retrying/skipping/finishing events to Slack/Hipchat, When `Notify all event to Slack/Hipchat` option is checked.

@cookpad/dev-infra 👓

### CODE REVIEW

**Comment by k0kubun** [comment]
For some slow tasks, I want to see only success events to check the results. In that case, launch event notifications are noisy for me since there is no need to check results when it's not finished. WDYT?

**Comment by eisuke** [comment]
@k0kubun I think it is not Kuroko2's responsibility. The notification (or message) like "Done, Please check my job 's results" is implemented in your batch jobs or workflows.

File: app/models/kuroko2/job_instance.rb [hunk 1]
```diff
@@ -43,10 +46,14 @@ def cancelable?
...
-  def cancel
+  def cancel(by:)
     self.tokens.destroy(*self.tokens)
     self.executions.destroy(*self.executions)
     self.touch(:canceled_at)
+
+    message = "This job was canceled by #{by}."
+    self.logs.warn(message)
+    Kuroko2.logger.warn(message)
   end
```
Comment (adorechic) [inline app/models/kuroko2/job_instance.rb]: Why don't you notify when a job is canceled manually?

Comment (eisuke) [inline app/models/kuroko2/job_instance.rb]: I judged it better to keep the same as before. 
Do you have a situation of necessity?

Comment (adorechic) [inline app/models/kuroko2/job_instance.rb]: You described this new option as "Notify **all** event to ...", so this suppressing is unexpected or confusing behavior for users, isn't it?

Comment (eisuke) [inline app/models/kuroko2/job_instance.rb]: >  confusing behavior for users

agree. And reconsider my opinion, So it changes to enable manual cancellation notifications https://github.com/cookpad/kuroko2/pull/8/commits/01cf47f16a027a9e16fbb07ec5cefddce6216c80

**Comment by adorechic** [comment]
LGTM

**Pull request merged**